pkg/transparentproxy/config/config_executables.go

```diff
@@ -468,9 +468,7 @@ func (c *Executables) InitializeIPv4(
 		return legacy, nil
 	case nft.Functionality.Rules.ExistingRules && legacy.Functionality.Rules.ExistingRules:
 		switch {
-		case nft.Functionality.Chains.DockerOutput && legacy.Functionality.Chains.DockerOutput:
-			fallthrough
-		case !nft.Functionality.Chains.DockerOutput && !legacy.Functionality.Chains.DockerOutput:
+		case nft.Functionality.Chains.DockerOutput && legacy.Functionality.Chains.DockerOutput, !nft.Functionality.Chains.DockerOutput && !legacy.Functionality.Chains.DockerOutput:
```

Comment:
I think the equivalent expression is 
```go
case nft.Functionality.Chains.DockerOutput == legacy.Functionality.Chains.DockerOutput:
```
Because basically we're trying to warn for the case when both are enabled or disabled